challenges-09/challenges-09.test.js

```diff
@@ -177,6 +183,9 @@ const isPrime = (value) => {
 
 const countPrimeNumbers = (arr) => {
   // Solution code here...
+  let count =0;
```

Comment:
You could have solved this without creating a variable outside of the `reduce` method, but this works!